templates/SimpleInventory/edit-board.html

```diff
@@ -0,0 +1,45 @@
+{% extends "SimpleBase/base.html" %}
+
+{% block head %}
+{{ block.super }}
+{% load static %}
+<link rel="stylesheet" href="{% static 'SimpleInventory/css/orderboard.css' %}">
+{% endblock %}
+
+{% block js %}
+<div id="message-toast" class="mdl-js-snackbar mdl-snackbar">
+  <div class="mdl-snackbar__text"></div>
+  <button class="mdl-snackbar__action" type="button"></button>
+</div>
+{{ block.super }}
+{% load static %}
+<script src="{% static 'SimpleInventory/js/edit-board.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div id="formCard" class="form-card mdl-card mdl-cell--8-col mdl-cell--2-offset mdl-shadow--2dp">
+    <div class="mdl-card__title">
+        <h2 class="mdl-card__title-text">Edit Board</h2>
+    </div>
+    <div class="mdl-card__supporting-text">
+        <form method="post" action="{% url 'api_v1_board_update' board_id %}" id="editBoardForm" class="mdl-cell--2-offset mdl-cell--8-col">
```

Comment:
Same deal with the mdl-cell--2-offset as the previous comment, this time just on the form.